lib/rules/padding-line-between-statements.js

```diff
@@ -537,40 +563,46 @@ module.exports = {
                     match(nextNode, configure.next);
 
                 if (matched) {
-                    return PaddingTypes[configure.blankLine];
+                    if (typeof configure.blankLine === "number") {
+                        return [configure.blankLine, PaddingTypes.always];
+                    }
+                    return [0, PaddingTypes[configure.blankLine]];
```

Comment:
This changes the return type signature of this function. Can we update the JSDoc block?